.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: Create a report to help us improve
+---
+
+<!--
+NOTICE: While GitHub is the preferred channel for reporting issues/feedback, this is not a mechanism for receiving support under any agreement or SLA. If you require immediate assistance, please use official support channels.
+-->
+
+<!--
+FOR BUGS RELATED TO THE SALEFORCE CLI, please use this repository: https://github.com/forcedotcom/cli/issues
+-->
+
+### Summary
+
+_Short summary of what is going on or to provide context_.
+
+### Steps To Reproduce:
+
+1.  This is step 1.
+1.  This is step 2. All steps should start with '1.'
+
+### Expected result
+
+_Describe what should have happened_.
+
+### Actual result
+
+_Describe what actually happened instead_.
+
+### Additional information
+
+_Feel free to attach a screenshot_.
+
+**VS Code Version**:

```

Comment:
Similarly, I'd take out stuff around VSCode and the CLI in this section. If the bug is coming to the plugin repo, it's not likely to be relevant information.